api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -128,14 +131,27 @@ public Profile getProfile(DbUser userLite) {
     final List<String> accessTierShortNames =
         accessTierService.getAccessTierShortNamesForUser(user);
 
-    final List<RenewableAccessModuleStatus> modulesStatus =
+    // renewableAccessModuleStatus is deprecated and will be replaced by accessModules.
+    final List<RenewableAccessModuleStatus> renewableAccessModuleStatus =
         userService.getRenewableAccessModuleStatus(userLite);
-
     final ProfileRenewableAccessModules renewableAccessModules =
         new ProfileRenewableAccessModules()
-            .modules(modulesStatus)
+            .modules(renewableAccessModuleStatus)
+            .anyModuleHasExpired(
+                renewableAccessModuleStatus.stream()
+                    .anyMatch(RenewableAccessModuleStatus::getHasExpired));
+
+    final List<AccessModuleStatus> accessModuleStatuses =
+        accessModuleService.getClientAccessModuleStatus(userLite);
+    final ProfileAccessModules accessModules =
+        new ProfileAccessModules()
+            .modules(accessModuleStatuses)
             .anyModuleHasExpired(
-                modulesStatus.stream().anyMatch(RenewableAccessModuleStatus::getHasExpired));
+                accessModuleStatuses.stream()
+                    .anyMatch(
+                        a ->
+                            (a.getExpirationEpochMillis() != null
+                                && Instant.now().toEpochMilli() > a.getExpirationEpochMillis())));
```

Comment:
use `clock.instant()` instead of `Instant.now()` - this makes test injection much easier